README.md

```diff
@@ -102,31 +102,30 @@ To create the test database, run:
 createdb cfdm_unit_test
 ```
 
-Create the required roles in the test database by running:
-
-```
-psql <<db-connection-parameters>> -f data/create_roles.sql
-```
-
 #### Create a development database
 Before you can run this project locally, you'll need a development database.
 
+
 To create the development database, run:
 
 ```
 createdb cfdm_test
 ```
 
-Load our sample data into the development database (`cfdm_test`) by running:
+Set the environment variable SQLA_SAMPLE_DB_CONN to point to this database, using:
 
 ```
-pg_restore --dbname cfdm_test --no-acl --no-owner data/subset.dump
-./manage.py refresh_materialized
+export SQLA_SAMPLE_DB_CONN=<psql:address-to-box>
```

Comment:
In this case, we should set it to `jdbc:postgresql://localhost:5432/cfdm_test` for working locally, correct?  If so it may help to just put this in there directly.  Also, doing this didn't work for me, but I suspect it's because I'm not including the username and password in it.

So instead, should it actually be `jdbc:postgresql://<username>:<password>@localhost:5432/cfdm_test`?

Reply:
D'oh still not working for me, even after doing that.  I keep seeing this message:

`ERROR: Invalid JDBC URL (should start with jdbc:) : None`

Which means the URL is not in the format that `to_jdbc_url` is expecting.  I thought we fixed this, though, so you didn't have to supply everything with just your local DB.

Reply:
I named it with a `SQLA_` prefix intending for it to be a `SQLAlchemy` URL, not a `jdbc` URL.

Reply:
Aha, that did it, thank you!